src/bot/services.py

```diff
@@ -1,24 +1,37 @@
+from src.api.request_models.request import Status
+from src.bot.main import create_bot
 from src.core import settings
 from src.core.db import models
-from src.bot.main import create_bot
 
 bot = create_bot().bot  # временная копия бота до миграции на webhooks
 
 
-async def send_approval_callback(user: models.User):
-    await bot.send_message(
-        chat_id=user.telegram_id, text=(f"Привет, {user.name} {user.surname}! Поздравляем, ты в проекте!")
-    )
-
-
-async def send_rejection_callback(user: models.User):
-    await bot.send_message(
-        chat_id=user.telegram_id,
-        text=(
+async def send_message_request_status_changed(user: models.User, status: str) -> None:
+    """Отправить сообщение о решении по заявке в telegram."""
+    text = f"Привет, {user.name} {user.surname}! Поздравляем, ты в проекте!"
+    if status is Status.DECLINED:
+        text = (
             f"К сожалению, на данный момент мы не можем зарегистрировать вас"
             f" в проекте. Вы можете написать на почту "
             f"{settings.ORGANIZATIONS_EMAIL}. Чтобы не пропустить актуальные"
             f" новости Центра \"Ломая барьеры\" - вступайте в нашу группу "
             f"{settings.ORGANIZATIONS_GROUP}"
-        ),
+        )
+    await bot.send_message(chat_id=user.telegram_id, text=text)

```

Comment:
```suggestion
    await bot.send_message(chat_id=user.telegram_id, text)

```
Так лучше, и даже так должно работать.
```suggestion
    await bot.send_message(user.telegram_id, text)
```
Интересно, есть ли какие-то проверки у блэк, чтобы ругаться на `text=text`?